var/spack/repos/builtin/packages/rpm/package.py

```diff
@@ -0,0 +1,136 @@
+    variant('openmp', default=True, description="OpenMP multithreading support (default).")
+    variant('docs', default=False, description='build documentation')
+
+    # Always required
+    depends_on('popt')
+
+    # Without this file patch, we don't detect lua
+    depends_on('lua+pc-file@5.3.5:')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+
+    # Enable POSIX.1e draft 15 file capabilities support
+    depends_on('libcap', when="+posix")
+    depends_on('berkeley-db@4.5:')
+
+    # Required for National Language Support, if not present autopoint error
+    depends_on('gettext')
+    depends_on('file')  # provides magic.h
+    depends_on('libarchive')
+
+    # suppot for cryptographic signatures
+    depends_on('gnupg', when="+gpg")
+
+    # cryptographic library to support digests and signatures
+    depends_on('libgcrypt', when='-openssl')
+    depends_on('openssl@1.0.2:', when='+openssl')
+
+    # RPM needs some database, ndb requires no extra dependencies but sqlite does
+    depends_on('sqlite@3.22.0:', when='+sqlite')
+
+    # Python 2.x support is being deprecated
+    depends_on('python@3.1:', when='+sqlite')
+
+    # compression support
+    depends_on('zlib')
+
+    # Desired to install these formats for use
+    depends_on('bzip2')
+    depends_on('gzip')
+    depends_on('xz')
+
+    # java jara dependency analysis (already requirement for lua)
+    depends_on('unzip', type='run')
+
+    # Documentation dependencies
+    depends_on('doxygen', type="build", when="+docs")
+
+    def autoreconf(self, spec, prefix):
+        bash = which('bash')
+        bash('./autogen.sh', '--noconfigure')
+
+    def configure_args(self):
+        spec = self.spec
+
+        args = ["--enable-ndb"]
+
+        pkg_config = os.path.join(spec['lua'].prefix.lib, "pkgconfig")
+        os.environ['PKG_CONFIG_PATH'] = pkg_config
+        os.putenv('PKG_CONFIG_PATH', pkg_config)
+
+        # cryptography library defaults to libgcrypt, but doesn't hurt to specify
+        if "+openssl" in spec:
+            args.append("--with-crypto=openssl")
+            tty.warning(openssl_warning)
+        else:
+            args.append("--with-crypto=libgcrypt")
+
+        # Default to ndb (no deps) if sqlite not wanted
+        if "+sqlite" in spec:
+            args.append("--enable-sqlite")
+        if "+bdb_ro" in spec:
+            args.append("--bdb-ro")
+
+        # Enable support for selinux
+        if "+selinux" in spec:
+            args.append('--with-selinux')
+        if "+python" in spec:
+            args.append("--enable-python")
+
+        # enable POSIX.1e draft 15 file capabilities support
+        if "+posix" in spec:
+            args.append('--with-cap')
+
+        # OpenMP multithreading support automatically enabled if C compiler has
+        # support for OpenMP version 4.5 or higher
+        if "-openmp" in spec:
```

Comment:
It should be the same, but almost all package s use `~`:
```suggestion
        if "~openmp" in spec:
```